src/pager.rs

```diff
@@ -40,16 +40,24 @@ impl PagerKind {
     fn from_bin(bin: &str) -> PagerKind {
         use std::path::Path;
 
-        match Path::new(bin)
-            .file_stem()
-            .map(|s| s.to_string_lossy())
-            .as_deref()
-        {
+        let file_stem = Path::new(bin).file_stem();
+
+        match file_stem.map(|s| s.to_string_lossy()).as_deref() {
             Some("bat") => PagerKind::Bat,
```

Comment:
I think we can remove this case then, and handle it generically? That would also allow us to piggy-back on existing tests so that we don't have to write new ones.

Reply:
Totally agree, that `bat` case should already be covered by checking the binary name against the pager binary.